test/fixtures/basic/package.json

```diff
@@ -4,6 +4,7 @@
     "run-series": "^1.1.1"
   },
   "devDependencies": {
+    "coffee-script": "^1.10.0",
```

Comment:
Oh read the commit, it's a test case? I think a programmatic test should be added then